src/actions/articles.js

```diff
@@ -47,6 +47,8 @@ export function addArticle(articleDetails) {
   };
 }
 
+export function saveArticle() {}
```

Comment:
I think we can still use `addArticle()` for this. Articles currently have a few fields like `title`, `body`, `author`, etc. We would just be adding another field _(`published` / `isPublished` maybe)_ to determine whether they are published or not. Changing that state should be as simple as calling `editArticle()` as well.

Back end and database changes are required for this change as well.